spacewalk/package/spacewalk.spec

```diff
@@ -161,10 +161,10 @@ Requires:       postgresql84-contrib
 %else
 Requires:       postgresql-contrib >= 8.4
 %endif
-Requires:       postgresql >= 8.4
-# we do not support postgresql versions > 9.6 yet
-Conflicts:      postgresql >= 9.7
-Conflicts:      postgresql-contrib >= 9.7
+Requires:       postgresql >= 9.6
+# we do not support postgresql versions > 10.x yet
+Conflicts:      postgresql >= 11.0
```

Comment:
I think we need to "Conflict: >= 11" only. AFAIK with 11.0 a version "11" would be ok, which is not true.